webpages/settings/index.html

```diff
@@ -281,35 +312,13 @@ <h1>{{ msg("settings") }}</h1>
     <div class="main">
       <div
         class="categories-block"
-        v-click-outside="closesidebar()"
+        v-click-outside="closesidebar"
```

Comment:
Why remove `()`?

Reply:
It's a vue event... a past PR broke it

Reply:
Confirmed it works now, didn't work before